examples/xcm/reserve_transfer.eg.ts

```diff
@@ -0,0 +1,223 @@
+/**
+ * @title XCM Reserve Asset Transfer
+ * @stability unstable
+ * @description Perform an XCM reserve asset transfer, in which two chains, which
+ * do not trust one another, rely on a third chain to store assets and facilitate
+ * the exchange.
+ */
+
+import * as Rococo from "@capi/rococo-dev-xcm"
+import * as Statemine from "@capi/rococo-dev-xcm/statemine"
+import * as Trappist from "@capi/rococo-dev-xcm/trappist"
+import { assert, assertNotEquals } from "asserts"
+import { $, alice as root, hex, Rune } from "capi"
```

Comment:
@harrysolovay for `sudo` calls I switched `alexa` to `alice (or root)` because the default sudo key for these runtimes is `alice` key